apps/builder/app/builder/features/style-panel/shared/style-instance-style-data.ts

```diff
@@ -0,0 +1,108 @@
+import { useStore } from "@nanostores/react";
```

Comment:
sounds like file could be just instance-style-data.ts

Reply:
we already have a file called style-info.ts, seems like this logic is of the same type and should go in there as well